app/api/__init__.py

```diff
@@ -28,12 +29,14 @@
 api.route(EventList, 'event_list', '/events')
 api.route(EventDetail, 'event_detail', '/events/<int:id>', '/tickets/<int:ticket_id>/event',
           '/microlocations/<int:microlocation_id>/event', '/social_links/<int:social_link_id>/event',
-          '/sponsors/<int:sponsor_id>/event', '/tracks/<int:track_id>/event', '/call_for_papers/<int:call_for_paper_id>/event')
+          '/sponsors/<int:sponsor_id>/event', '/tracks/<int:track_id>/event',
+          '/event_copyright/<int:copyright_id>/event', '/call_for_papers/<int:call_for_paper_id>/event')
```

Comment:
This makes complete sense I think because this basically gives the URL for the endpoint to get the event that is related to a particular event_copyright. If /event_copyright/<int:copyright_id> makes sense then this also makes sense.